ercc/main.go

```diff
@@ -11,8 +11,10 @@ import (
 
 	"github.com/hyperledger/fabric-chaincode-go/shim"
 	"github.com/hyperledger/fabric-contract-api-go/contractapi"
-	"github.com/hyperledger/fabric-private-chaincode/ercc/attestation"
 	"github.com/hyperledger/fabric-private-chaincode/ercc/registry"
+	"github.com/hyperledger/fabric-private-chaincode/internal/attestation"
```

Comment:
This clearly shows that we have 2 attestation API, because the Go-based one uses that in `internal` rather than the `attestation-api`. Is there a way that we can find a path towards consolidation?

Reply:
What would you like to propose? For me, the current separation into a go impl and a c/c++ impl is ok.